docker/images/outrigger/README.md

```diff
@@ -0,0 +1,56 @@
+# Outrigger Docker Image
+
+# Building
+
+To build simply run:
+
+```
+docker build -t outrigger .
+``` 
+
+in the directory this README is currently in.
+
+# Running
+
+To run the outrigger server simply run:
+
+```
+docker run -d -p 8080:8080 -v `pwd`/juttles:/opt/juttles --name outrigger outrigger
```

Comment:
Alternatively outrigger should (does?) support a command line option to
pass the path to the config file.


On Mon, Jan 4, 2016 at 5:18 PM, Rodney Lopes Gomes <notifications@github.com
> wrote:

> In docker/images/outrigger/README.md
> <https://github.com/juttle/outrigger/pull/6#discussion_r48803671>:
>
> > +# Building
> > +
> > +To build simply run:
> > +
> > +```
> > +docker build -t outrigger .
> > +```
> > +
> > +in the directory this README is currently in.
> > +
> > +# Running
> > +
> > +To run the outrigger server simply run:
> > +
> > +```
> > +docker run -d -p 8080:8080 -v `pwd`/juttles:/opt/juttles --name outrigger outrigger
>
> I think there's a way to do this by documenting to share the right
> external path to an internal path such as /opt/outrigger so the working
> directory is always set to that and we'd load any juttle-config.js or
> juttle-config.json present there. Let me try this out locally and
> document accordingly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/juttle/outrigger/pull/6/files#r48803671>.
>



-- 
Michael Demmer
Jut - VP Engineering
demmer@jut.io